src/get_changes.js

```diff
@@ -0,0 +1,29 @@
+import * as core from "@actions/core"
+
+// Get list of changed files
+export async function getChangedFiles(githubClient, options, context) {
+	if (!options.commit || !options.baseCommit) {
+		core.setFailed(
+			`The base and head commits are missing from the payload for this ${context.eventName} event.`,
+		)
+	}
+
+	// Use GitHub's compare two commits API.
+	// https://developer.github.com/v3/repos/commits/#compare-two-commits
+	const response = await githubClient.repos.compareCommits({
```

Comment:
Would it be possible to do this using git locally? We supposedly have the whole git history available already, so going to an API (which might fail) seem silly.

Reply:
Thanks for looking through the changes!
Yeah that's a really good point. I just took this from https://github.com/jitterbit/get-changed-files/blob/master/src/main.ts#L62, but it is overkill as you say. Will look into improving with a simple `git diff` 👍

Reply:
Ok actually, I just tried this on a branch, and realised that actions/checkout@v2 seems to only fetch a single commit by default. So running a local git diff produces the error:
```
fatal: bad object <commit>
```
I then tried updating my checkout to
```
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
```
and the command succeeded, but I don't think it would be a good idea to require this. So it was a good idea, and I think it may have worked with actions/checkout@v1, but probably not viable sorry.

Reply:
OK, after thinking about it, it probably does make sense to check it in the API, since we want to avoid creating too much dependencies for users to have to set up. In that case this PR looks good!